src/MudBlazor/Components/DatePicker/MudBaseDatePicker.razor.cs

```diff
@@ -275,7 +281,7 @@ protected IEnumerable<DateTime> GetWeek(int month, int index)
                 yield return week_first.AddDays(i);
         }
 
-        private string GetWeekNumber(int month, int index)
+        public string GetWeekNumber(int month, int index)
```

Comment:
```suggestion
        private string GetWeekNumber(int month, int index)
```

Why did you change this to public?

Reply:
to enable units tests on it. Do you have any other way to do that?

Reply:
Ah, I see. You could change it to `internal`. Ideally, you would open the picker and test if the correct week is displayed in the dom.

Reply:
ok, I changed public to internal. I will look at dom checks on the way to do it